tests/test_e2e_14_mef_eline.py

```diff
@@ -59,11 +66,35 @@ def create_evc(self, uni_a="00:00:00:00:00:00:00:01:1", uni_z="00:00:00:00:00:00
                 "tag": {"tag_type": 1, "value": vlan_id}
             }
         }
+        if primary_path:
+            payload["primary_path"] = primary_path
+            payload["dynamic_backup_path"] = False
+        if backup_path:
+            payload["backup_path"] = backup_path
         api_url = KYTOS_API + '/mef_eline/v2/evc/'
         response = requests.post(api_url, json=payload)
         data = response.json()
         return data['circuit_id']
 
+    def get_evc_data(self, evc_id:str) -> dict:
+        api_url = KYTOS_API + '/mef_eline/v2/evc/' + evc_id
+        response = requests.get(api_url)
+        return response.json()
+    
+    def redeploy_evc(self, evc_id, try_avoid_same_s_vlan=True):
+        str_avoid_vlan = "false"
+        if try_avoid_same_s_vlan:
+            str_avoid_vlan = "true"
+        api_url = f"{KYTOS_API}/mef_eline/v2/evc/{evc_id}/redeploy?try_avoid_same_s_vlan={str_avoid_vlan}"
+        requests.patch(api_url)
```

Comment:
Let's also assert a successful request response code here, just so if one day this returns a 400 or whatever else it won't be difficult to figure out

Reply:
Added assertion for the responses.